development-strategy.md

```diff
@@ -73,6 +85,7 @@ __BACKEND__
 * Access to `db`... Check the user if exists or not.
 * Write table user as online.
 *  Build `route`. Send confirmation/error `response` (`Successfully signed in`).
+* (optional) if user is a student , he sees the tests directly in the main section. So he easily continue to solve.
```

Comment:
@feridunAKYOL I think only one main page all the clients, if we make route another page for registered user he can't see home page after signing in. But we can let him go to his page by clicking his name. And in my opinion it will also be easy to manage for us in this way.

Reply:
you're right. I deleted this line